docs/source/tutorials/quickstart.rst

```diff
@@ -177,19 +182,139 @@ The output should look as follows.
+relative to each other.
+
+For more information on ``next_to`` and other positioning methods, check out the
+list of :class:`.Mobject` methods in our reference manual.
+
+
+Using ``.animate`` syntax to animate methods
+*************************************************
+
+The final lesson in this tutorial is using ``.animate``, a mobject method which
+animates changes you make to a mobject. When you prepend ``.animate`` to any
+method call that modifies a mobject, the method becomes an animation which
+can be played using ``self.play``. Let's return to ``SquareToCircle`` to see the
+differences between using methods when creating a mobject,
+and animating those method calls with ``.animate``.
+
+1. Open ``scene.py``, and add the following code snippet below the ``SquareAndCircle`` method:
+
+.. code-block:: python
+
+   class AnimatedSquareToCircle(Scene):
+       def construct(self):
+           circle = Circle()  # create a circle
+           square = Square()  # create a square
+
+           self.play(Create(square))  # show the shapes on screen
+           self.play(square.animate.rotate(PI / 4))  # rotate the square
+           self.play(
+               ReplacementTransform(square, circle)
+           )  # transform the square into a circle
+           self.play(
+               circle.animate.set_fill(PINK, opacity=0.5)
+           )  # color the circle on screen
+
+2. Render ``AnimatedSquareToCircle`` using the following command:
+
+.. code-block:: bash
+
+   manim -pql scene.py AnimatedSquareToCircle
+
+The following animation displays:
+
+.. manim:: AnimatedSquareToCircle2
+   :hide_source:
+
+   class AnimatedSquareToCircle2(Scene):
+       def construct(self):
+           circle = Circle()  # create a circle
+           square = Square()  # create a square
+
+           self.play(Create(square))  # show the shapes on screen
+           self.play(square.animate.rotate(PI / 4))  # rotate the square
+           self.play(ReplacementTransform(square, circle))  # transform the square into a circle
+           self.play(circle.animate.set_fill(PINK, opacity=0.5))  # color the circle on screen
+
+The first ``self.play`` creates the square. The second animates rotating it 45 degrees.
+The third transforms the square into a circle, and the last colors the circle pink.
+Although the end result is the same as that of ``SquareToCircle``, ``.animate`` shows
+``rotate`` and ``set_fill`` applying to the mobject dynamically, instead of creating them
+with the changes already applied.
+
+Try other methods, like ``flip`` or ``shift``, and see what happens.
+
+Note that ``.animate`` transforms the mobject, regardless of the mobject method you use.
+For example, ``rotate`` doesn't actually rotate the square. Instead, manim takes the starting state
+(the square) and the ending state (the square, rotated 45 degrees), and interpolates
+the two states. That is, instead of rotating the square, it transforms the square into another, rotated square.
```

Comment:
Maybe add a short explanation of how this might result in weird results for large angles, and to use `Rotate` instead for proper rotation, similar to how https://try.manim.community/ explains it.

Reply:
I'm adding two steps that use ``DifferentRotations`` from the link you provided to talk about the quirks of ``.animate``, though it will probably need a review to make sure I got it right.